src/sankey.js

```diff
@@ -184,6 +184,11 @@ export default function() {
     }
 
     function initializeNodeDepth() {
+      var L = max(nodesByBreadth, function(nodes) {
+        return nodes.length;
+      });
+      var maxNodePadding = 2/3 * size[1] / (L - 1);
```

Comment:
I like the 2/3 - looks good. Lets define it up top though by the properties block:

https://github.com/plotly/d3-sankey/blob/6f19390be0da4a81abf6f7f6cfdf5fc8dc73e583/src/sankey.js#L7-L11

not that I think we need to make it a property now, though we may want to do that later. For now I just want it to be easier to find, in case someone's wondering why they get our warning (or why the padding refuses to get bigger)

Reply:
Thank you for pointing this out! It should now be resolved in commit 1a79abb!